### PR TITLE
feat: Support env in mysql

### LINKED ISSUE #533: make katib-db configurable
https://github.com/kubeflow/katib/blob/3cbb5654042cf73b164685a0cf6f2f30e6286c8b/pkg/db/v1alpha2/interface.go#L21

Please make `katib-db:3306` configurable with  os.Getenv("MYSQL_HOST") and os.Getenv("MYSQL_PORT")

### PR DESCRIPTION
**Pull request by gaocegege**
Signed-off-by: Ce Gao <gaoce@caicloud.io>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #533 

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/katib/868)

### CODE REVIEW

File: pkg/db/v1alpha3/common/helper.go [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package common
+
+import "os"
+
+func GetEnvWithDefault(name, defaultValue string) string {
```
Comment (johnugeorge) [inline pkg/db/v1alpha3/common/helper.go]: https://github.com/kubeflow/katib/blob/master/pkg/controller.v1alpha3/consts/const.go#L37

Comment (johnugeorge) [reply pkg/db/v1alpha3/common/helper.go]: already implemented

Comment (gaocegege) [reply pkg/db/v1alpha3/common/helper.go]: Gotcha, going to reuse it.

**Pull request merged**